README.md

```diff
@@ -105,8 +105,10 @@ optional arguments:
                         hand, this may cause a long delay to gather all
                         results.
   --print-found         Do not output sites where the username was not found.
-  --no-color            Don't color terminal output
-  --browse, -b          Browse to all results on default bowser.
+  --no-color            Don\'t color terminal output
```

Comment:
Was the escaped single quote here intentional?

Reply:
No, it was accidentally added. It seems to break markdown highlight in my editor.